safe/impact_functions/base.py

```diff
@@ -1408,7 +1408,10 @@ def _calculate_impact(self):
         start_time = datetime.now()
 
         # Run the IF. self.run() is defined in each IF.
-        result_layer = self.run()
+        try:
+            result_layer = self.run()
+        except KeywordNotFoundError as e:
+            raise KeywordNotFoundError(e)
```

Comment:
Don't you want `raise` only ? If you raise a exception, you will loose the previous traceback in the `run()`